andela/neon-ah-backend
# #163288966 Change id types to UUIDV4

#### What does this PR do?
Change all model id's to type UUID

#### Description of Task to be completed?
Change all model id's from INTEGER to type UUID

#### How should this be manually tested?
 • clone the repo
 • cd into the repo
 • [Setup development database](https://github.com/andela/neon-ah-backend/pull/14#issue-244353384)
 • run `npm install`
 • run `npm run migrate && npm run seed`
 • Check the id columns of the seeded users, roles and authTypes tables.

#### What are the relevant pivotal tracker stories?
[Change id types to UUID](https://www.pivotaltracker.com/story/show/163288966)
